From 70c89c1b0b89e302e54adc93ff643975987b00b0 Mon Sep 17 00:00:00 2001
From: Konrad Adamczyk <kadamczyk@marvell.com>
Date: Thu, 6 Sep 2018 12:05:54 +0200
Subject: [PATCH 0829/1239] octeontx: svc: change id of SMC calls

Change id of Octeontx SMC calls to match SiP Services range
from ARM DEN0028B.

Signed-off-by: Konrad Adamczyk <kadamczyk@marvell.com>
---
 .../arm/include/asm/arch-octeontx/octeontx_svc.h | 14 +++++++-------
 .../include/asm/arch-octeontx2/octeontx2_svc.h   | 16 ++++++++--------
 2 files changed, 15 insertions(+), 15 deletions(-)

diff --git a/arch/arm/include/asm/arch-octeontx/octeontx_svc.h b/arch/arm/include/asm/arch-octeontx/octeontx_svc.h
index 72dde37c1a..5827b63310 100644
--- a/arch/arm/include/asm/arch-octeontx/octeontx_svc.h
+++ b/arch/arm/include/asm/arch-octeontx/octeontx_svc.h
@@ -10,18 +10,18 @@
 
 /* SMC function IDs for general purpose queries */
 
-#define OCTEONTX_SVC_CALL_COUNT		0x4300ff00
-#define OCTEONTX_SVC_UID		0x4300ff01
+#define OCTEONTX_SVC_CALL_COUNT		0xc200ff00
+#define OCTEONTX_SVC_UID		0xc200ff01
 
-#define OCTEONTX_SVC_VERSION		0x4300ff03
+#define OCTEONTX_SVC_VERSION		0xc200ff03
 
 /* OcteonTX Service Calls version numbers */
-#define OCTEONTX_VERSION_MAJOR	0x0
-#define OCTEONTX_VERSION_MINOR	0x2
+#define OCTEONTX_VERSION_MAJOR	0x1
+#define OCTEONTX_VERSION_MINOR	0x0
 
 /* x1 - node number
  */
-#define OCTEONTX_DRAM_SIZE		0x43000301
-#define OCTEONTX_NODE_COUNT		0x43000601
+#define OCTEONTX_DRAM_SIZE		0xc2000301
+#define OCTEONTX_NODE_COUNT		0xc2000601
 
 #endif /* __OCTEONTX_SVC_H__ */
diff --git a/arch/arm/include/asm/arch-octeontx2/octeontx2_svc.h b/arch/arm/include/asm/arch-octeontx2/octeontx2_svc.h
index 38eae6c9af..79e88781ce 100644
--- a/arch/arm/include/asm/arch-octeontx2/octeontx2_svc.h
+++ b/arch/arm/include/asm/arch-octeontx2/octeontx2_svc.h
@@ -10,19 +10,19 @@
 
 /* SMC function IDs for general purpose queries */
 
-#define OCTEONTX2_SVC_CALL_COUNT		0x4300ff00
-#define OCTEONTX2_SVC_UID		0x4300ff01
+#define OCTEONTX2_SVC_CALL_COUNT		0xc200ff00
+#define OCTEONTX2_SVC_UID		0xc200ff01
 
-#define OCTEONTX2_SVC_VERSION		0x4300ff03
+#define OCTEONTX2_SVC_VERSION		0xc200ff03
 
 /* OcteonTX Service Calls version numbers */
-#define OCTEONTX2_VERSION_MAJOR	0x0
-#define OCTEONTX2_VERSION_MINOR	0x3
+#define OCTEONTX2_VERSION_MAJOR	0x1
+#define OCTEONTX2_VERSION_MINOR	0x0
 
 /* x1 - node number
  */
-#define OCTEONTX2_DRAM_SIZE		0x43000301
-#define OCTEONTX2_NODE_COUNT		0x43000601
-#define OCTEONTX2_DISABLE_RVU_LFS	0x43000b01
+#define OCTEONTX2_DRAM_SIZE		0xc2000301
+#define OCTEONTX2_NODE_COUNT		0xc2000601
+#define OCTEONTX2_DISABLE_RVU_LFS	0xc2000b01
 
 #endif /* __OCTEONTX2_SVC_H__ */
-- 
2.29.0

